tests/aws/services/sns/test_sns.py

```diff
@@ -265,7 +268,13 @@ def test_publish_by_path_parameters(
         queue_url = sqs_create_queue()
         sns_create_sqs_subscription(topic_arn=topic_arn, queue_url=queue_url)
 
-        client = aws_http_client_factory("sns", region="us-east-1")
+        client = aws_http_client_factory(
+            "sns",
+            signer_factory=SigV4Auth,
+            region=TEST_AWS_REGION_NAME,
+            aws_access_key_id=TEST_AWS_ACCESS_KEY_ID,
+            aws_secret_access_key=TEST_AWS_SECRET_ACCESS_KEY,
+        )
```

Comment:
This ensures that the credentials are correctly sent when using the API in a raw manner.

Reply:
q: can this work against AWS? the test is AWS validated

Reply:
Yes. I also sneaked a small change in 534348be9 to ensure multi-region

Reply:
Okay great, I wasn't sure, thanks for confirming! 😄